fabgonzalez-mt-vernon.md

```diff
@@ -0,0 +1,9 @@
+# Mt. Vernon, OH
+
+### Places to eat
+- Fiesta Mexicana
+- Southside West
+
+### Things to do
+- Camping
```

Comment:
@DieselAnalytics Not really, this is a very small town. Just sports from local high schools. :disappointed: